SourceryTests/SourcerySpec.swift

```diff
@@ -1271,14 +1294,15 @@ class SourcerySpecTests: QuickSpec {
                         }.toNot(throwError())
                 }
 
+#if canImport(ObjectiveC)
```

Comment:
This is related to crash in Swift compiler [reported here](https://github.com/apple/swift/issues/68296).